bgpd/bgp_route.h

```diff
@@ -327,6 +327,7 @@ struct bgp_path_info {
 #define BGP_PATH_ACCEPT_OWN (1 << 16)
 #define BGP_PATH_MPLSVPN_LABEL_NH (1 << 17)
 #define BGP_PATH_MPLSVPN_NH_LABEL_BIND (1 << 18)
+#define BGP_PATH_UNSORTED (1 << 19)
```

Comment:
Does it mean: that if `bgp_path_info` is not evaluated yet as the best path, it's marked as UNSORTED, otherwise - SORTED. But it's not related to the actual sorting of the paths in `bgp_dest`?

Reply:
yes this is just a tag that tracks whether or not a path_info has had best path run over it at all..  Unsorted was just choosen as a name because I think of it that way